tardis/opacities/opacity_solver.py

```diff
@@ -46,7 +46,9 @@ def solve(self, legacy_plasma) -> OpacityState:
                         legacy_plasma.atomic_data.lines.shape[
                             0
                         ],  # number of lines
-                        legacy_plasma.abundance.shape[1],  # number of shells
+                        legacy_plasma.number_density.shape[
```

Comment:
Be careful when we merge the macroatom as there are a couple small changes to this file, though I don't think it should conflict

Reply:
or the other way round - this should be merged first.